src/components/PartyReservation.tsx

```diff
@@ -16,100 +16,93 @@ interface PartyReservationProps {
 // refactor below to use TS to extend Model.Guest but include only the boolean types
 interface RadioButtonKeys {
   is_attending: string;
-  is_vaccinated: string
+  is_vaccinated: string;
 }
 
 interface OptionType {
   label: string;
   value: string;
 }
 
-const Container = tw.div`mt-28`;
-const RadioContainer = tw.div`mt-2`;
+const Container = tw.div``;
+const InputLabel = tw(Paragraph)`uppercase text-sm mb-0`;
+const RadioContainer = tw.div`flex flex-row`;
 const SubmitButton = tw.button`h-12 w-full border rounded border-dark-gray mt-3 font-urbanist uppercase font-bold hover:bg-green`;
-
-const GuestContainer = styled(Paragraph)`
-    ${tw`text-xl mb-6`}
-`
-
-const FormRow = styled.div`
-  ${tw`flex flex-row w-full`}
-  & > div:last-of-type {
-    ${tw`ml-3 w-full flex-1`}
-  }
-`;
-
-const RadioButtonSelectionContainer = tw.div``
+const GuestContainer = tw(Paragraph)`text-xl mb-6`;
+const FormRow = tw.div`flex flex-col w-full`;
+const Section = tw.div`flex flex-row items-center justify-between mb-3`;
+const CustomSelect = tw(Select)`uppercase text-sm`;
 
 const mealOptions: OptionType[] = [
   { value: 'steak', label: 'Steak' },
   { value: 'chicken', label: 'Chicken' },
   { value: 'salmon', label: 'Salmon' },
 ];
 
-const PartyReservation = ({
-  party,
-  guests,
-}: PartyReservationProps) => {
-
+const PartyReservation = ({ party, guests }: PartyReservationProps) => {
   const dispatch = useDispatch();
-  const [partyData, updatePartyData] =
-    useState<Models.Party>(party);
-  const [guestsData, updateGuestsData] =
-    useState<Models.Guest[]>(guests);
-
-  const booleanRadioChangeHandler = (guestId: Models.Guest['id'], guestBooleanProperty: string, checked: boolean) => {
-    const guest = { ...guestsData.find((g) => g.id === guestId) } as Models.Guest
+  const [partyData, updatePartyData] = useState<Models.Party>(party);
+  const [guestsData, updateGuestsData] = useState<Models.Guest[]>(guests);
+
+  const booleanRadioChangeHandler = (
+    guestId: Models.Guest['id'],
+    guestBooleanProperty: string,
+    checked: boolean
+  ) => {
+    const guest = {
+      ...guestsData.find((g) => g.id === guestId),
+    } as Models.Guest;
     if (guest.id == null) return;
     // guest.is_vaccinated = checked
     // type WritableKeys<T> = {
```

Comment:
We can remove these comments. I left them here as a reminder to try and ask you. What I was trying to do on line 61 was something like: `guest[guestBooleanProperty as keyof Models.Guest] = checked;`, or at least I think that's what it was. But I was getting an error because `id` is a readonly property. Found some stuff online to maybe get around that, but the typescript was too confusing for me haha. So I made that `RadioButtonKeys` interface. If you know of a way to get only the writeable attributes of Models.Guest, and if you think that's a better approach then another interface, please change it! 😄